samples/drivers/adc/boards/mec172xevb_assy6906.overlay

```diff
@@ -6,6 +6,19 @@
 
 / {
 	zephyr,user {
-		io-channels =  <&adc0 0>, <&adc0 3>, <&adc0 4>, <&adc0 5>;
+		io-channels =  <&adc0 3>;
```

Comment:
Why not provide configuration for the other channels instead of removing them?

Reply:
Because other boards only test one channel...
But your suggestion is good. This test case can support multiple channels.
I'll add all these channels.